base_layer/core/src/proof_of_work/monero_rx/helpers.rs

```diff
@@ -105,8 +105,11 @@ pub fn verify_header(
 ) -> Result<MoneroPowData, MergeMineError> {
     let monero_data = MoneroPowData::from_header(header, consensus)?;
     let expected_merge_mining_hash = header.merge_mining_hash();
-    let extra_field = ExtraField::try_parse(&monero_data.coinbase_tx_extra)
-        .map_err(|_| MergeMineError::DeserializeError("Invalid extra field".to_string()))?;
+    let extra_field = ExtraField::try_parse(&monero_data.coinbase_tx_extra);
```

Comment:
Is there no other data about the error here? 
Maybe create an issue to add more data about what went wrong